app/src/main/java/org/simple/clinic/patient/shortcode/FilterDigits.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.patient.shortcode
+
+class FilterDigits: UuidShortCodeCreator.CharacterFilter {
```

Comment:
This class name should be on the same lines as `CharacterFilter`.
Can you rename it to `DigitFilter`

Reply:
Done.